_posts/2021-08-22-release-2.0.0.md

```diff
@@ -0,0 +1,162 @@
+
+### File formats
+
+- Fixes support for DL_POLY HISTORY files that contain cell information even if there are no periodic boundary conditions (Issue [#3314][])
+- PDBWriter will use chainID instead of segID (Issue [#3144][])
+- PDBParser and PDBWriter now assign and use the element attribute (Issues [#3030][] [#2422][])
+
+### Analyses
+
+- Fixes issue when atttempting to use/pass mean positions to PCA analysis (Issue [#2728][])
+- Fixes issue with WaterBridgeAnalysis double counting waters (Issue [#3119][])
+- Documents and fixes the `density` keyword for `rdf.InterRDF_s` (Isuse [#2811][])
+- Fixed Janin analysis residue filtering, including CYSH (Issue [#2898][])
+
+## Changes to functionality
+- Deprecated `hbonds.hbond_analysis` has been removed in favour of `hydrogenbonds.hbond_analysis` (Issues [#2739][], [#2746][])
+- TPRParser now loads TPR files with `tpr_resid_from_one=True` by deafult, which starts TPR resid indexing from 1 (instead of 0 as in previous MDAnalysis versions) (Issue [#2364][], PR [#3152][])
+- `analysis.hole` has now been removed in favour of `analysis.hole2.hole` (Issue [#2739][])
+- `Writer.write(Timestep)` and `Writer.write_next_timestep` have been removed. Please use `write()` instead (Issue [#2739][])
+- Removes deprecated density_from_Universe, density_from_PDB, Bfactor2RMSF, and notwithin_coordinates_factory from MDAnalysis.analysis.density (Issue [#2739][])
+- Removes deprecated `waterdynamics.HydrogenBondLifetimes` (PR [#2842][])
+- `hbonds.WaterBridgeAnalysis` has been moved to `hydrogenbonds.WaterBridgeAnalysis` (Issue [#2739][] PR [#2913][])
+
+## Deprecations
+- The `bfactors` attribute is now aliased to `tempfactors` and will be removed in 3.0.0 (Issue [#1901][])
+- `WaterBridgeAnalysis.generate_table()` now returns table information, with the `table` attribute being deprecated
+- Various analysis result attributes which are now stored in `Results` will be deprecated in 3.0.0 (Issue [#3261][])
+- In 3.0.0 the ParmEd classes will only be accessible from the `MDAnalysis.converters` module
+- In 2.1.0 the TRZReader will default to a dt of 1.0 ps when failing to read it from the input TRZ trajectory
+
+## Author statistics
+
+Altogether, this represents the work of 35 authors. 17 of which were new contributors:
+
+ - @daveminh
+ - @mieczyslaw
+ - @aditya-kamath
+ - @tirkarthi
+ - @LeonardoBarneschi
+ - @hejamu
+ - @biogen98
+ - @orioncohen
+ - @z3y50n
+ - @hp115
+ - @ojeda-e
+ - @thadanipaarth
+ - @HenryKobin
+ - @1ut
+ - @sulays
+ - @ahy3nz
+ - @fiszczyp
+
+MDAnalysis thanks [NumFOCUS][]'s continued support as the organisation's fiscale sponsor.
+
+- The MDAnalysis Team
+
+[roadmap]: {{ site.baseurl }}{% post_url 2019-11-06-roadmap %}
+[conda-forge channel]: https://anaconda.org/conda-forge/mdanalysis
+[PyPi]: https://pypi.org/project/MDAnalysis/
+[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
```

Comment:
I realise we've been neglecting our userguide responsibilities in favour of the main repo. I'm switching to stable from 2.0.0 for now. I assume we can release this and then work on bumping up the user guide afterwards.

Does that work for you @lilyminium ?

Reply:
Yeah, I've been pretty neglectful as well. There's a handful of issues to address for 2.0, not hard, just need a little bit of sitting down and writing. We can get that out fast.

Reply:
Please do ping us to do stuff, it's a collective responsibility of all coredevs to make sure we have a functioning userguide.